packages/api/src/command/medical/patient/settings/create-patient-settings.ts

```diff
@@ -26,6 +27,14 @@ type PatientSettingsUpsertProps = PatientSettingsUpsertForCxProps & {
   patientIds: string[];
 };
 
+type BatchProcessingResult = {
+  processedCount: number;
+  failedCount: number;
+  error?: Error;
```

Comment:
I suggest we treat those as `unknown`. Almost sure we're getting Error, but in JS/TS it's valid to send other types as errors.